handlers/search.js

```diff
@@ -91,28 +95,36 @@ function searchPage(request, reply) {
   const filters = getFilters(query);
   const advancedSearch = query.advanced_search || (Object.keys(filters).length > 0);
 
-  trials.search(queryStr, page, perPage, filters).then((_trials) => {
-    const currentPage = page || 1;
-    const pagination = generatePaginationLinks(
-      request.url, currentPage,
-      perPage, maxPages,
-      _trials.total_count
-    );
-    const queryDescription = queryStr || 'clinical trial data';
-    const title = `Search results for ${queryDescription} in OpenTrials`;
-    const description = `OpenTrials has ${_trials.total_count} matches for
+  sources.list().then((_sources) => {
+    trials.search(queryStr, page, perPage, filters).then((_trials) => {
+      const currentPage = page || 1;
+      const pagination = generatePaginationLinks(
+        request.url, currentPage,
+        perPage, maxPages,
+        _trials.total_count
+      );
+      const queryDescription = queryStr || 'clinical trial data';
+      const title = `Search results for ${queryDescription} in OpenTrials`;
+      const description = `OpenTrials has ${_trials.total_count} matches for
                         ${queryDescription} in its clinical trials database`;
-
-    reply.view('search', {
-      title,
-      description,
-      query,
-      currentPage,
-      pagination,
-      advancedSearchIsVisible: advancedSearch,
-      trials: _trials,
+      const sourcesList = _sources.filter((s) => (
+        ['register', 'journal'].indexOf(s.type) > -1)
+      );
```

Comment:
The formatting here is quite confusing. I think this could either be in a single line, or move the closing parenthesis in line 111 to 112, so:

```javascript
const sourcesList = _sources.filter((s) => (
  ['register', 'journal'].indexOf(s.type) !== -1
));
```

(I also prefer checking `!== -1` than `> -1`, but there's no argument for/against each style, as they both have the same result.)